coalib/results/Diff.py

```diff
@@ -438,9 +436,15 @@ def change_line(self, line_nr, original_line, replacement):
                                                replacement)
             replacement = ''.join((orig_diff + new_diff).modified)
 
-        linediff.change = (original_line, replacement)
+        linediff.change = (self._file[line_nr-1], replacement)
         self._changes[line_nr] = linediff
 
+    def change_line(self, line_nr, original_line, replacement):
+        # Use of change_line method is deprecated
+        logging.warning('Use of orginal_line argument is deprecated. '
```

Comment:
the warning doesn't help me, if I see this I'd remove the original line parameter and everything would break. The method is deprecated, not the parameter. Sorry, can't merge this. Secondly you're using a warning, we agreed at some point that we wanted to use `debug` for this because it affects developers, not users